pkgs/games/katago/default.nix

```diff
@@ -86,6 +87,8 @@ stdenv.mkDerivation rec {
     "-DUSE_TCMALLOC=ON"
   ] ++ lib.optionals enableBigBoards [
     "-DUSE_BIGGER_BOARDS_EXPENSIVE=ON"
+  ] ++ lib.optionals enableTrtPlanCache [
+    "-DUSE_CACHE_TENSORRT_PLAN=ON"
```

Comment:
Since we are here, use `lib.cmake*` functions (you can find them in the file lib/strings.nix)

Reply:
Adjusted, thanks for the heads up, I was unaware that these existed.